odata-json-format/16 Bound Function.md

```diff
@@ -337,6 +337,11 @@ how to interpret the omitted parameter. Note: a nullable non-binding
 parameter is equivalent to being annotated as optional with a default
 value of `null`.
 
+Non-binding collection-valued parameters MAY be omitted from the request body regardless
+of nullability or optionality. Unless a `Core.OptionalValue` with `DefaultValue` is annotated,
```

Comment:
DefaultValue has type String, so it only works for primitive single-valued :-(

Reply:
We could extend the term `Core.OptionalParameter` with `AnyDefaultValue` or `NonPrimitiveDefaultValue` or similar of type `Edm.Untyped`.